### PR TITLE
DO NOT MERGE: ODK

### PR DESCRIPTION
**Pull request by matentzn**
ODK config is managed [here](https://github.com/monarch-initiative/monarch-ontology/issues/18)

### CODE REVIEW

**Comment by matentzn** [comment]
I could use some help with the Definition of Done here; even if it is just a brain dump. @cmungall @kshefchek 
By that I mean: in the absence of the possibility of reviewing the entire diff; what needs to happen so we can merge this in? As a first pass I would say this:
- [ ] Make list of and review MAJOR differences (i.e. 1000K UBERON subs missing or similar)
- [ ] Make list of terms in old monarch-ontology not anymore in new
- [ ] Make list of terms without labels
- [X] Ontology consistent

What else would you say? Is this enough to merge in, and then fix the almost certainly upcoming issues as we go along, or are there other things we need to look out for?

**Comment by cmungall** [comment]
Whatever we do, it needs to be timed with whatever the monarch data release cycle is, so there are no surprises.

Semantics-based QC is hard due to knock-on effect of inference. We need to get some kind of system in place, I started writing something.

For now, can you summarize in a few sentences what you think the *expected* changes are based on the changes you made.
